dll/win32/setupapi/devinst.c

```diff
@@ -2,7 +2,7 @@
  * SetupAPI device installer
  *
  * Copyright 2000 Andreas Mohr for CodeWeavers
- *           2005-2006 Herv� Poussineau (hpoussin@reactos.org)
+ *           2005-2006 Hervé Poussineau (hpoussin@reactos.org)
```

Comment:
Whitespace change.

Reply:
I can't fix it, sorry. For me it seems there is no difference between these two lines.

Reply:
It's not a whitespace change, it's an encoding change:

![firefox_2018-01-04_16-05-47](https://user-images.githubusercontent.com/685851/34569472-299db13e-f169-11e7-989a-9b9ddce87408.png)

Reply:
It seems the new encoding is UTF-8, which is perfectly OK.

Reply:
![image](https://user-images.githubusercontent.com/578406/34574099-747fc4c6-f187-11e7-80ec-c946f8c76f1a.png)

Interesting... for me it looks like this. Maybe browser use some workaround?

Reply:
It looks like that in github, that's why I pasted a screenshot of the email.

Reply:
Ahrg, changed it now to WIN-1252 (last commit "Whitspace fix") ... @HBelusca . I've readed your comment after that.